photon-client/src/integTest/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriverIntegTest.scala

```diff
@@ -77,7 +77,7 @@ class GameTrainingDriverIntegTest extends SparkTestUtils with GameTestUtils with
   def testFixedEffectsWithIntercept(): Unit = sparkTest("testFixedEffectsWithIntercept", useKryo = true) {
 
     // This is a baseline RMSE capture from an assumed-correct implementation on 01/24/2018
-    val errorThreshold = 1.2
+    val errorThreshold = 2.4
```

Comment:
Each of these test values has been multiplied by a factor of `2` - but that's incorrect.

Since the old test values were set for the old (incorrect) computation `RMSE = sqrt(MSE / 2)`, we need to correct these test values. But the correct factor is not `2`, it's `sqrt(2)`, since we need to undo the `sqrt` first.

Reply:
Changed to the correct threshold in PR 442